### DISCUSSION TITLE
Overriding checkbox options leads to invalid submit

### DISCUSSION THREAD

**Discussion by mprofitlich** [verbb/formie#1557] [Q&A] [answered]
We overwrite all available options of a checkbox field. In the backend we just setup a single placeholder option as Formie asks for at least one option.

When submitting the form, we receive an error messag: [Field Label] is invalid.

We checked: The HTML of the inputs corresponds with the HTML that formie outputs when not overwriting the field options.

Reading the [docs](https://verbb.io/craft-plugins/formie/docs/template-guides/rendering-fields#override-field-settings) we understand it is possible to set available field options via Twig, overwriting the ones set in the backend.

Does someone have a hint how to trace the error? 

```
{% set workshopsArray = [] %}
{% for entry in workshops %}
    {% for workshop in entry.inhalte.type('workshop') %}
        {% if workshop.title == craft.app.request.post("title") %}
            {% set isTrue = true %}
        {% else %}
            {% set isTrue = false %}
        {% endif %}
        {% set workshopsArray = workshopsArray|merge(
            [
                {
                    label: workshop.title,
                    value: workshop.title|lower,
                    isDefault: isTrue
                }
            ]
        ) %}
    {% endfor %}
{% endfor %}

{% do form.setFieldSettings('workshops', {
    options: workshopsArray
}) %}
{{ craft.formie.renderForm('contactform') }}
```

**Comment by engram-design** [comment, marked as answer]
Can't seem to replicate this, sorry. Can you confirm things are working with a simplified test?

![image](https://github.com/verbb/formie/assets/1221575/7bd33fae-befd-499d-8882-e4a08466e1c1)

```twig
{% do form.setFieldSettings('checkboxes', {
    options: [
        { label: 'Override 1', value: 'override1' },
        { label: 'Override 2', value: 'override2', isDefault: true },
        { label: 'Override 3', value: 'override3', isDefault: true },
    ],
}) %}
```

![image](https://github.com/verbb/formie/assets/1221575/ba549cfe-8025-4b72-ad14-47d4db0f0946)

**Reply by mprofitlich** [reply]
Thanks for the test you did. We followed your setup in a simple environment (outside the project we want to use it) and it works indeed. Thus the problem arises from something else. I’ll dive into it and I close this question as the functionality is given as expected.

Thank you, Josh!
